src/main/resources/templates/admin/category/add.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="admin/template :: head(title = 'EduQuiz - Admin')">
+</head>
+<body class="hold-transition sidebar-mini">
+<div class="wrapper">
+
+    <th:block th:replace="admin/template :: navbar" />
```

Comment:
Use the complete syntax of fragment expressions
```suggestion
    <th:block th:replace="~{admin/template :: navbar}" />
```